<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log</class>
 <widget class="QMainWindow" name="log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIGNUP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 255);
font: 25pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="reg_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(255, 85, 255);
background-color: rgb(0, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgb(200, 0, 245);
border-radius: 7px;
width: 200 px;
height: 45px;
}
QPushButton:hover {
background-color: rgba(5, 251, 255, 200);
}
QPushButton:pressed {
background-color: rgba(0, 255, 255, 65);
}</string>
    </property>
    <property name="text">
     <string>Авторизоваться</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLineEdit" name="password_save">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="name_save">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
